src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -265,10 +259,6 @@ export default class AdvancedFilters extends Vue {
     );
   }
 
-  beforeMount() {
-    this._chiMajorVersion = detectMajorChiVersion();
```

Comment:
This removes the lifecycle hook but the declaration of `_chiMajorVersion` is still there. Please check line 73.